opal/runtime/opal_progress.c

```diff
@@ -262,45 +262,6 @@ int opal_progress_set_event_flag(int flag)
     return tmp;
 }
 
-void opal_progress_event_users_increment(void)
```

Comment:
Because of the TCP BTL, this code is still needed.

Reply:
Done